.github/workflows/build_navitia_packages_for_release.yml

```
@@ -2,6 +2,8 @@ name: Build Navitia Packages For Release
 
 on:
   push:
+    tags:
+      - '*'
     branches:
       - release
```

```suggestion
      - '*'
    branches:
      - dev
```
If we use github to make releases, it means that the `release` branch will become useless, right ? If yes, then I think we  need to run this workflow when new tags are pushed to the `dev` branch

---

Yes, totally right. But I intentionally keep everything in place just in case, before testing this new workflow. Once done and tested, I'll prepare another PR to clean up all of that stuff.